databrowser/src/components/select/SelectCustom.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <select
+    v-model="value"
+    class="
+      py-1
+      pr-6
+      pl-4
+      rounded-full
+      focus:border-green-500
+      appearance-none
+      outline-none
+      bg-white bg-no-repeat
+      border border-gray-500
+    "
+  >
+    <slot></slot>
+  </select>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/runtime-core';
+
+export default defineComponent({
+  props: {
+    modelValue: {
+      required: false,
+      default: () => null,
+      type: null,
```

Comment:
What does type `null` mean? Shouldn't it be `String`?

Reply:
`null` in this case means arbitrary type, because the value could be anything (of course, most of the time it will be a string, but could also be a number). We could explicitly state the types like `[String, Number, Object, ...]` or leave it as null, what do you say?

Reply:
Oh I see! Then it's fine I guess.